src/Feedback-Service/Dtos/CreateFeedbackDto.cs

```diff
@@ -1,8 +1,25 @@
+using System.ComponentModel.DataAnnotations;
+using System.Runtime.InteropServices;
+
 namespace Feedback_Service.Dtos;
 
-public class CreateFeedbackDTO
+public class CreateFeedbackDto
 {
-    public Guid PatientId { get; set; }
+    public int PatientId { get; set; }
+    public Guid AuthorId { get; set; }
     public Guid StressMeassurementId { get; set; }
-    public string Feedback { get; set; } = "";
-}
\ No newline at end of file
+    public string? FeedbackComment { get; set; }
+}
+
+// using System.ComponentModel.DataAnnotations;
+// using System.Runtime.InteropServices;
+
+// namespace Feedback_Service.Dtos;
+
+// public record CreateFeedbackDto
+// (
+//     [Required] int PatientId,
+//     [Required] Guid AuthorId,
+//     [Optional] Guid? StressMeassurementId,
+//     [Optional] string? FeedbackComment
+// );
```

Comment:
Is there a way we can keep this as a CreateFeedbackDto record?

Reply:
yesss, just resolved it